### DISCUSSION TITLE
How to execute in local machine backend.ai

### DISCUSSION THREAD

**Discussion by Jeffr-K** [lablup/backend.ai#1331] [Q&A] [answered]
There is a lot of repo in backend.ai repository. Can I get some information what is essential repositories to execute in local such as frontend and backend?

**Comment by agatha197** [comment, marked as answer]
@Jeffr-K You can use backend.ai's `install-dev.sh` script for backend. Here is the [guide](https://github.com/lablup/backend.ai#installation-for-single-node-development). As you can see through the guide, you can use `--editable-webui` option or clone the [backend.ai-webui](https://github.com/lablup/backend.ai-webui) and follow [this one](https://github.com/lablup/backend.ai-webui#developing--testing-without-bundling).
